internal/pkg/config/v1/context_state.go

```diff
@@ -19,6 +21,20 @@ type ContextState struct {
 	Nonce            []byte      `json:"nonce,omitempty"`
 }
 
+func (c *ContextState) GetAuth() *AuthConfig {
```

Comment:
This function already exists for `Context`!

Reply:
Oh right, I'll switch it around.